labels.md

```diff
@@ -114,8 +114,6 @@ particular issue. See more in [project management document](project-management.m
   to tackle them.
 - performance
   Anything affecting performance metrics.
-- refactoring
```

Comment:
do not agree

Reply:
It was supplementary, not an issue type. And it's just "enhancement + I4".